lib/shared/sdk/usbboot.js

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2016 Raspberry Pi Foundation
```

Comment:
I wonder if this Copyright line should be moved under the https://github.com/raspberrypi/usbboot link?
\*shrug\* I'm not a Lawyer ;-)